<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>516</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>281</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="workWithOneQuant_RB">
       <property name="text">
        <string>Квантерион №1 + первое число в стеке</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="workWithTwoQuant_RB">
       <property name="text">
        <string>Первая колонка + запись в стек</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="plusBtn">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minusBtn">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="multBtn">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="divBtn">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>23</y>
      <width>571</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <widget class="QLineEdit" name="firstQuant_I_LE">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>70</y>
            <width>218</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>4</y>
            <width>199</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Кватернион №1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>8</x>
            <y>108</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>j</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="firstQuant_K_LE">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>140</y>
            <width>218</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="firstQuant_J_LE">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>105</y>
            <width>218</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>8</x>
            <y>38</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>r</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>8</x>
            <y>73</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>i</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="firstQuant_R_LE">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>35</y>
            <width>218</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>8</x>
            <y>143</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>k</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>180</y>
            <width>279</width>
            <height>367</height>
           </rect>
          </property>
          <widget class="QLineEdit" name="secondQuant_I_LE">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>70</y>
             <width>218</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>16</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>i</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>192</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Кватернион №2</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="secondQuant_R_LE">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>35</y>
             <width>218</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="secondQuant_K_LE">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>140</y>
             <width>218</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="secondQuant_J_LE">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>105</y>
             <width>218</width>
             <height>29</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>16</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>j</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>16</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>k</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>16</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>r</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="stack_F_N_W" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <widget class="QLabel" name="label_25">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>k</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="firstStackQuant_R_LE">
          <property name="geometry">
           <rect>
            <x>32</x>
            <y>33</y>
            <width>218</width>
            <height>29</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_22">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>36</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>r</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="firstStackQuant_I_LE">
          <property name="geometry">
           <rect>
            <x>32</x>
            <y>68</y>
            <width>218</width>
            <height>29</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_23">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>71</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>i</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_24">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>105</y>
            <width>16</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>j</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="firstStackQuant_K_LE">
          <property name="geometry">
           <rect>
            <x>32</x>
            <y>138</y>
            <width>218</width>
            <height>29</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_21">
          <property name="geometry">
           <rect>
            <x>8</x>
            <y>2</y>
            <width>196</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Первый элемент в стеке</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="firstStackQuant_J_LE">
          <property name="geometry">
           <rect>
            <x>32</x>
            <y>103</y>
            <width>218</width>
            <height>29</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QWidget" name="stack_S_N_W" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>180</y>
            <width>259</width>
            <height>176</height>
           </rect>
          </property>
          <widget class="QLineEdit" name="secondStackQuant_K_LE">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>140</y>
             <width>218</width>
             <height>29</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="secondStackQuant_I_LE">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>70</y>
             <width>218</width>
             <height>29</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_29">
           <property name="geometry">
            <rect>
             <x>8</x>
             <y>107</y>
             <width>16</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>j</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_28">
           <property name="geometry">
            <rect>
             <x>8</x>
             <y>72</y>
             <width>16</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>i</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_27">
           <property name="geometry">
            <rect>
             <x>8</x>
             <y>37</y>
             <width>16</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>r</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="secondStackQuant_R_LE">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>35</y>
             <width>218</width>
             <height>29</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_26">
           <property name="geometry">
            <rect>
             <x>6</x>
             <y>4</y>
             <width>193</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Второй элемент в стеке</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_30">
           <property name="geometry">
            <rect>
             <x>8</x>
             <y>143</y>
             <width>16</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>k</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="secondStackQuant_J_LE">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>105</y>
             <width>218</width>
             <height>29</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
